lib/rpcc_dpdk/rte_flow_ndn.c

```diff
@@ -842,3 +842,71 @@ rpc_rte_flow_release_united_items(rcf_rpc_server       *rpcs,
 
     RETVAL_VOID(rte_flow_release_united_items);
 }
+
+int
+rpc_rte_flow_indir_action_conf_create(rcf_rpc_server *rpcs,
+                                      const asn_value *asn_conf,
+                                      rpc_rte_flow_indir_action_conf_p *conf)
```

Comment:
Why do we need so complicated handling of rte_flow_indir_action_conf whereas the structure is really simple in fact?